scripts/paddle.gd

```diff
@@ -34,9 +34,20 @@ func expand_pad():
 	if (get_parent().get_active_powerups().has("ExpandPad")):
 		scale_x_by(1)
 
-func _input(event):
-	if !ball_presence and (event.type == InputEvent.MOUSE_BUTTON and event.is_pressed()):
+func spawn_balls(num_balls):
+	for i in range(num_balls):
 		var ball = ball_scene.instance()
 		ball.set_pos(get_pos() - Vector2(0, 32))
```

Comment:
It would be cool to have balls spawn from other balls, and in different directions.